### ISSUE TITLE
Inconsistent handling of `--version` by Dhall executables

### ISSUE THREAD

**Issue by vvv** [dhall-lang/dhall-haskell#1331]
`dhall-to-json` and `dhall-to-yaml` react to `--version` differently, which is surprising.
```
$ dhall-to-json --version
1.4.0
$ dhall-to-yaml --version
Invalid option `--version'

Usage: dhall-to-yaml [--explain] [--omitNull | --omitEmpty] [--documents]
                     [--quoted] [[--key ARG] [--value ARG] | --noMaps]
                     [--file FILE]
  Compile Dhall to YAML
```

Ideally, all Dhall executables should understand `--version` (they do understand `--help` after all).
```
$ dhall --version
Invalid option `--version'

Usage: dhall [version | resolve | type | normalize | repl | diff | hash | lint |
               format | freeze | encode | decode | text | [--file FILE] 
               [--annotate] [--alpha] [--no-cache]] [--explain] [--plain] 
             [--ascii]
  Interpreter for the Dhall language
$ yaml-to-dhall --version
Missing: SCHEMA

Usage: yaml-to-dhall [-V|--version] SCHEMA [--records-strict]
                     [--no-keyval-arrays] [--no-keyval-maps] [--unions-first |
                     --unions-none | --unions-strict] [--file FILE] [--ascii]
                     [--plain]
  Convert a YAML expression to a Dhall expression, given the expected Dhall type
$ json-to-dhall --version
Missing: SCHEMA

Usage: json-to-dhall [-V|--version] SCHEMA [--records-strict]
                     [--no-keyval-arrays] [--no-keyval-maps] [--unions-first |
                     --unions-none | --unions-strict] [--file FILE] [--ascii]
                     [--plain]
  Convert a JSON expression to a Dhall expression, given the expected Dhall type
```

**Comment by sjakobi** [comment]
I agree that `--version` should be handled consistently. I'll make a patch.

**Comment by Gabriel439** [comment]
Huh, I thought I fixed this in https://github.com/dhall-lang/dhall-haskell/pull/1199

**Comment by sjakobi** [comment]
> Huh, I thought I fixed this in #1199

You did fix it for the executables in dhall-json. I'll also fix it for the other executables in this project.

**Comment by vvv** [comment]
> Huh, I thought I fixed this in #1199

@Gabriel439 Whoops, I forgot about that.
